war/src/main/less/modules/table.less

```diff
@@ -49,16 +49,22 @@
             width: 24px;
           }
         }
+
+        svg {
+          vertical-align: middle;
+          width: 0.8rem;
+          height: 0.8rem;
+        }
```

Comment:
In particular, this is the hunk that fixes [JENKINS-70369](https://issues.jenkins.io/browse/JENKINS-70369). Could possibly be backported to `stable-2.375` if desired, but likely not worth the effort for such a minor issue.